### PR TITLE
Fix performance tuning

### PR DESCRIPTION
**Pull request by michalkulakowski**

### CODE REVIEW

**Review by dtrawins — CHANGES_REQUESTED** [review]
There are more out of data info in this page. For example old parameters which are deprecated now.

File: docs/performance_tuning.md [hunk 1]
```diff
@@ -113,21 +112,20 @@ In case of using CPU plugin to run the inference, it might be also beneficial to
 
 | Parameters      | Description |
 | :---        |    :----   |
-| CPU_THREADS_NUM       | Specifies the number of threads that CPU plugin should use for inference.     |
-| CPU_BIND_THREAD   |   Binds inference threads to CPU cores.      |
-| CPU_THROUGHPUT_STREAMS | Specifies number of CPU "execution" streams for the throughput mode |
+| INFERENCE_NUM_THREADS       | Specifies the number of threads that CPU plugin should use for inference.     |
```
Comment (dtrawins) [inline docs/performance_tuning.md:115]: there is no param INFERENCE_NUM_THREADS

**Review by dtrawins — COMMENTED** [review]
It would be good to include a recommendation to use REST with KServe raw input

**Review by rasapala — APPROVED** [review]
LGTM

**Review by dtrawins — APPROVED** [review]

**Pull request merged**